roles/matrix-postgres/tasks/import_generic_sqlite_db.yml

```diff
@@ -0,0 +1,84 @@
+---
+
+# Pre-checks
+
+- name: Fail if Postgres not enabled
+  fail:
+    msg: "Postgres via the matrix-postgres role is not enabled (`matrix_postgres_enabled`). Cannot import."
+  when: "not matrix_postgres_enabled|bool"
+
+- name: Fail if playbook called incorrectly
+  fail:
+    msg: "The `sqlite_database_path` variable needs to be provided to this playbook, via --extra-vars"
+  when: "sqlite_database_path is not defined or sqlite_database_path.startswith('<')"
+
+- name: Fail if playbook called incorrectly
+  fail:
+    msg: >-
+      The `postgres_db_connection_string` variable needs to be provided to this playbook, via `--extra-vars`.
+      Example: `--extra-vars="postgres_db_connection_string=postgresql://username:password@localhost:<port>/database_name`"
+  when: "postgres_db_connection_string is not defined or not postgres_db_connection_string.startswith('postgresql://')"
```

Comment:
We could provide that to the user, we should know the `connString` of `matrix-postgres` already, and avoid people potentially having their db-password in their bash_history

Reply:
You're right! This is quite raw right now. We should provide a way to just pass a "service name" and have the connection string extracted out of `matrix_postgres_additional_databases`.

That is, if we're staying with such a manual-migration setup.

Reply:
Fixed in 5dba0c038b6c19a9b.

The import command is something like this now:

```
ansible-playbook -v -i inventory/hosts setup.yml \
--tags=import-generic-sqlite-db \
--extra-vars='sqlite_database_path=/matrix/appservice-discord/data/discord.db postgres_connection_string_variable_name=matrix_appservice_discord_database_connString'
```